inf-clojure.el

```diff
@@ -384,6 +384,13 @@ mode line entirely."
   :type 'sexp
   :risky t)
 
+(defvar inf-clojure-eldoc-enabledp t
+  "Var that allows disabling `eldoc-mode` in `inf-clojure`.
+
+Set to `nil` to disable eldoc.  Eldoc can be quite useful by
+displaying funciton signatures in the modeline, but can also
```

Comment:
Btw, this disclaimer is true for every command that does some code evaluation on the side. :-)

Reply:
That's true, but all others are user driver. If you refresh a namespace or call doc, you understand that `*1` and others will have been moved because you caused an evaluation. Eldoc is special because unbeknownst to the user those vars will change value. It makes using them very frustrating.

Reply:
Also - there's a typo in `function`.